src/sky.cpp

```diff
@@ -202,58 +203,132 @@ void Sky::render()
 
 		driver->setMaterial(m_materials[1]);
 
-		video::SColor cloudyfogcolor = m_bgcolor;
+		if (m_use_old_skybox) {
+			// Draw middle band of the sky, unsmoothed:
```

Comment:
Please keep tjhe original comments for the classic skybox:
`// Draw far cloudy fog thing blended with skycolor`